logging/src/test/resources/logging.xml

```diff
@@ -151,6 +151,17 @@
         <pattern-formatter pattern="%d{HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n" color-map="info:cyan,warn:yellow,error:red"/>
     </formatter>
 
+    <formatter name="JSON">
+        <json-formatter date-format="yyyy-MM-dd'T'HH:mm:ssSSS" pretty-print="true" print-details="true" zone-id="GMT">
+            <exception-output-type value="detailed-and-formatter"/>
```

Comment:
typo: should be `detailed-and-formatted`

Reply:
why is the test passing? The EXCEPTION_OUTPUT_TYPE attribute uses `detailed-and-formatted` in its allowed values, it should have rejected the `detailed-and-formatter` value

Reply:
That's a good question. I can see the error being thrown because of the typo, but the test controller seems to be booting fine. I'll have to dig into this.

Reply:
It looks like this produced a runtime failure and the test controller doesn't boot with `rollbackOnRuntimeFailure` set to `true`. So the error was hidden.